source/change-streams.rst

```diff
@@ -282,28 +305,36 @@ Driver API
      */
     collation: Optional<Document>;
 
+    /**
+     * The changeStream will only provides changes that occurred after the
+     * specified cluster timestamp. The default value is a cluster time obtained from
+     * the server before the changestream was created.
```

Comment:
Should "changestream" be written as "change stream" for consistency with other parts of the spec? Both here and two lines up.

Reply:
done